src/Analyzers/Core/Analyzers/UseCollectionInitializer/AbstractUseCollectionInitializerAnalyzer.cs

```diff
@@ -47,13 +47,18 @@ internal abstract class AbstractUseCollectionInitializerAnalyzer<
         TVariableDeclaratorSyntax,
         TAnalyzer>, new()
 {
+    public readonly record struct AnalysisResult(
+        ImmutableArray<Match> PreMatches,
+        ImmutableArray<Match> PostMatches);
```

Comment:
similarly, we have a few of these floating around.  i'd like to get it down to 1.